### PR TITLE
test(change_detection): add an integration test verifying that binding p...

### PR DESCRIPTION
**Pull request by vsavkin**
...ropagation config works

### CODE REVIEW

File: modules/angular2/test/core/compiler/integration_spec.js [hunk 1]
```diff
@@ -198,6 +199,26 @@ export function main() {
           done();
         })
       });
+
+      it("should provide binding configuration config to the component", (done) => {
```
Comment (yjbanov) [inline modules/angular2/test/core/compiler/integration_spec.js]: use consistent quotes

File: modules/angular2/test/core/compiler/integration_spec.js [hunk 2]
```diff
@@ -198,6 +199,26 @@ export function main() {
...
+        compiler.compile(MyComp, el('<push-cmp #cmp></push-cmp>')).then((pv) => {
```
Comment (yjbanov) [inline modules/angular2/test/core/compiler/integration_spec.js]: do you not need to return the promise from the test function?

Comment (yjbanov) [inline modules/angular2/test/core/compiler/integration_spec.js]: nvm, it's the "done" trick, isn't it?

File: modules/angular2/test/core/compiler/integration_spec.js [hunk 3]
```diff
@@ -214,8 +235,34 @@ class MyDir {
...
+  bpc:BindingPropagationConfig;
+
+  constructor(bpc:BindingPropagationConfig) {
+    this.numberOfChecks = 0;
+    this.bpc = bpc;
+    bpc.shouldBePropagated();
+  }
+
+  get field(){
+    this.numberOfChecks ++;
```
Comment (yjbanov) [inline modules/angular2/test/core/compiler/integration_spec.js]: remove space before ++

**Comment by vicb** [comment]
91fd5a69bf43ab1a9188305f41136cea7b70eb9f

**Pull request closed without merging**